bingwallpaper

```diff
@@ -28,6 +28,20 @@ size="1920x1200"
 # Collection Path.
 path="$HOME/Pictures/Bing/"
 
+# Make it run just once (useful to run as a cron)
+run_once=false
```

Comment:
yeah, but the install package sets up a startup service as well. We need to add that to the readme.